2023-12-11 00:20:46.104363+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "MIGRATION_MODEL" not found (this database is empty); SQL statement:
SELECT ID, VERSION FROM MIGRATION_MODEL ORDER BY UPDATE_TIME DESC [42104-224]
2023-12-11 00:21:12.179947+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "DATABASECHANGELOG" not found (this database is empty); SQL statement:
SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOG [42104-224]
2023-12-11 00:21:14.821948+01:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "DATABASECHANGELOGLOCK" not found (this database is empty); SQL statement:
SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOGLOCK [42104-224]
2023-12-11 00:21:14.989950+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "DATABASECHANGELOG" non trouvée
Table "DATABASECHANGELOG" not found; SQL statement:
SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOG [42102-224]
2023-12-11 01:00:52.218246+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de données est déjà fermée (pour désactiver la fermeture automatique à l'arrêt de la VM, ajoutez "; DB_CLOSE_ON_EXIT = FALSE" à l'URL db)
Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1425)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:485)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:511)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:455)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:787)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:196)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:454)
	at io.quarkus.agroal.runtime.DataSources_Bean.doDestroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:82)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:472)
	at io.quarkus.arc.Arc.shutdown(Arc.java:66)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:53)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:436)
2023-12-11 11:11:40.370629+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de données est déjà fermée (pour désactiver la fermeture automatique à l'arrêt de la VM, ajoutez "; DB_CLOSE_ON_EXIT = FALSE" à l'URL db)
Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1425)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:485)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:511)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:455)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:787)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:196)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:454)
	at io.quarkus.agroal.runtime.DataSources_Bean.doDestroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:82)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:472)
	at io.quarkus.arc.Arc.shutdown(Arc.java:66)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:53)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:227)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:71)
	at org.keycloak.quarkus.runtime.KeycloakMain.start(KeycloakMain.java:117)
	at org.keycloak.quarkus.runtime.cli.command.AbstractStartCommand.run(AbstractStartCommand.java:33)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2026)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
	at picocli.CommandLine.execute(CommandLine.java:2170)
	at org.keycloak.quarkus.runtime.cli.Picocli.parseAndRun(Picocli.java:119)
	at org.keycloak.quarkus.runtime.KeycloakMain.main(KeycloakMain.java:107)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.doRun(QuarkusEntryPoint.java:61)
	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.main(QuarkusEntryPoint.java:32)
2023-12-11 11:11:40.442633+01:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de données est déjà fermée (pour désactiver la fermeture automatique à l'arrêt de la VM, ajoutez "; DB_CLOSE_ON_EXIT = FALSE" à l'URL db)
Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1425)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:485)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:511)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:455)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:787)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:196)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:454)
	at io.quarkus.agroal.runtime.DataSources_Bean.doDestroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:82)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:472)
	at io.quarkus.arc.Arc.shutdown(Arc.java:66)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:53)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:227)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:71)
	at org.keycloak.quarkus.runtime.KeycloakMain.start(KeycloakMain.java:117)
	at org.keycloak.quarkus.runtime.cli.command.AbstractStartCommand.run(AbstractStartCommand.java:33)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2026)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
	at picocli.CommandLine.execute(CommandLine.java:2170)
	at org.keycloak.quarkus.runtime.cli.Picocli.parseAndRun(Picocli.java:119)
	at org.keycloak.quarkus.runtime.KeycloakMain.main(KeycloakMain.java:107)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.doRun(QuarkusEntryPoint.java:61)
	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.main(QuarkusEntryPoint.java:32)
2023-12-11 11:49:39.670450+01:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLTimeoutException: Dépassement du temps lors du vérrouillage de la table {0}
Timeout trying to lock table {0}; SQL statement:
insert into REALM (ACCESS_CODE_LIFESPAN,LOGIN_LIFESPAN,USER_ACTION_LIFESPAN,ACCESS_TOKEN_LIFESPAN,ACCESS_TOKEN_LIFE_IMPLICIT,ACCOUNT_THEME,ADMIN_EVENTS_DETAILS_ENABLED,ADMIN_EVENTS_ENABLED,ADMIN_THEME,ALLOW_USER_MANAGED_ACCESS,BROWSER_FLOW,CLIENT_AUTH_FLOW,DEFAULT_LOCALE,DEFAULT_ROLE,DIRECT_GRANT_FLOW,DOCKER_AUTH_FLOW,DUPLICATE_EMAILS_ALLOWED,EDIT_USERNAME_ALLOWED,EMAIL_THEME,ENABLED,EVENTS_ENABLED,EVENTS_EXPIRATION,INTERNATIONALIZATION_ENABLED,LOGIN_THEME,LOGIN_WITH_EMAIL_ALLOWED,MASTER_ADMIN_CLIENT,NAME,NOT_BEFORE,OFFLINE_SESSION_IDLE_TIMEOUT,OTP_POLICY_ALG,OTP_POLICY_DIGITS,OTP_POLICY_COUNTER,OTP_POLICY_WINDOW,OTP_POLICY_PERIOD,OTP_POLICY_TYPE,PASSWORD_POLICY,REFRESH_TOKEN_MAX_REUSE,REGISTRATION_ALLOWED,REG_EMAIL_AS_USERNAME,REGISTRATION_FLOW,REMEMBER_ME,RESET_CREDENTIALS_FLOW,RESET_PASSWORD_ALLOWED,REVOKE_REFRESH_TOKEN,SSL_REQUIRED,SSO_IDLE_TIMEOUT,SSO_IDLE_TIMEOUT_REMEMBER_ME,SSO_MAX_LIFESPAN,SSO_MAX_LIFESPAN_REMEMBER_ME,VERIFY_EMAIL,ID) values (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?) [50200-224]
2023-12-11 23:31:11.030938+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de données est déjà fermée (pour désactiver la fermeture automatique à l'arrêt de la VM, ajoutez "; DB_CLOSE_ON_EXIT = FALSE" à l'URL db)
Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1425)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:485)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:511)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:455)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:787)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:196)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:454)
	at io.quarkus.agroal.runtime.DataSources_Bean.doDestroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:82)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:472)
	at io.quarkus.arc.Arc.shutdown(Arc.java:66)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:53)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:436)
